### PR TITLE
Deprecate distribution pairs

### PR DESCRIPTION
**Pull request by jschueller**
Using array accessors is easier, from both c++ and Python apis,
in order to define an Histogram or UserDefined distribution.

### CODE REVIEW

File: lib/src/Uncertainty/Distribution/openturns/Histogram.hxx [hunk 1]
```diff
@@ -111,6 +111,12 @@ public:
   void setFirst(const NumericalScalar first);
   NumericalScalar getFirst() const;
 
+  /** Data accessor */
+  void setData(const NumericalPoint & l,
+               const NumericalPoint & h);
+  NumericalPoint getL() const;
```
Comment (dbarbier) [inline lib/src/Uncertainty/Distribution/openturns/Histogram.hxx]: Why l and h member names, and not width/height?

Comment (jschueller) [inline lib/src/Uncertainty/Distribution/openturns/Histogram.hxx]: @dbarbier, ok, i'll rename the attributes

**Comment by regislebrun** [comment]
+1

**Comment by jschueller** [comment]
@dbarbier, @sofianehaddad, any thoughts here ?

**Comment by jschueller** [comment]
@dbarbier, @sofianehaddad, any thoughts here ?

**Comment by jschueller** [comment]
@dbarbier, @sofianehaddad, any thoughts here ?

**Comment by sofianehaddad** [comment]
seems ok, +1

**Pull request merged**